test/apiv2/20-containers.at

```diff
@@ -63,7 +63,7 @@ cid=$(jq -r '.Id' <<<"$output")
 # Prior to the fix in #6835, this would fail 500 "args must not be empty"
 t POST   libpod/containers/${cid}/start '' 204
 # Container should exit almost immediately. Wait for it, confirm successful run
-t POST   libpod/containers/${cid}/wait  '' 200 '0'
+t POST   libpod/containers/${cid}/wait?condition=stopped&condition=exited  '' 200 '0'
```

Comment:
I think that libpod is sending decimal numbers (enum vals) now (podman v3), because `ToParams()` is called. If this should work we would need also to update code generator.

Reply:
The change to generator would be easy, but since podman v3 is out I don't know if that's good idea. To be clear I like descriptive string value much more than plain ints.

Reply:
Or maybe we shouldn't use generate `ToParams()` here at all, and revert it to manual uri parameter creation.

Reply:
Thanks for the suggestion, I've updated the generator here: https://github.com/containers/podman/pull/9414/commits/1e8f31bd16c831f13ba7c92447d99c8e0321fa9d.

Reply:
> please put the URI into quotes
> `t POST "libpod/containers/${cid}/wait?condition=stopped&condition=exited" '' 200 '0'`

@edigaryev  It will fix the CI fail, the `&` causes troubles if not withing "".